src/wp-includes/formatting.php

```diff
@@ -5847,36 +5847,32 @@ function wp_spaces_regexp() {
 }
 
 /**
- * Prints the important emoji-related styles.
+ * Enqueue the important emoji-related styles.
  *
- * @since 4.2.0
+ * @since 6.4.0
  */
-function print_emoji_styles() {
-	static $printed = false;
-
-	if ( $printed ) {
-		return;
+function enqueue_emoji_styles() {
```

Comment:
Since this is a new function being introduced, should we take this opportunity to give it a `wp_` prefix?